Aplib.Core/Tactics/FirstOfTactic.cs

```diff
@@ -21,19 +21,19 @@ public FirstOfTactic(List<Tactic> subTactics) : base(subTactics)
         /// </summary>
         /// <param name="subTactics">The list of sub-tactics.</param>
         /// <param name="guard">The guard condition.</param>
-        public FirstOfTactic(List<Tactic> subTactics, Func<bool> guard) : base(subTactics, guard)
+        public FirstOfTactic(Func<bool> guard, params Tactic[] subTactics) : base(guard, subTactics)
         {
         }
 
         /// <inheritdoc/>
-        public override List<PrimitiveTactic> GetFirstEnabledActions()
+        public override Action? GetAction()
         {
             foreach (Tactic subTactic in SubTactics)
             {
-                List<PrimitiveTactic> firstOfTactics = subTactic.GetFirstEnabledActions();
+                Action? action = subTactic.GetAction();
 
-                if (firstOfTactics.Count > 0)
-                    return firstOfTactics;
+                if (action is not null && action.IsActionable())
```

Comment:
If the Action from the subTactic was not actionable, it is null, right? So it can just be:
```suggestion
                if (action is not null)
```